codex-ui/src/vue/components/tabbar/Tab.vue

```diff
@@ -26,9 +27,11 @@
           />
         </div>
       </template>
-      {{ title }}
+      <div :class="[$style['tab__body-text']]">
+        {{ title }}
+      </div>
       <Icon
-        v-if="closable"
+        v-if="closable && isNarrow"
```

Comment:
Maybe rename isNarrow to more clear?

Reply:
any suggestions? (this variable checks that cross icon could be placed in available space)

Reply:
maybe `isFit`?

Reply:
isNarrow now deprecated